src/pages/regulatory/_eu-grid.tsx

```diff
@@ -85,67 +85,67 @@ const EUgrid = () => {
         <CssGrid columns="repeat(2, 1fr)">
             <CssGridColumn>
                 <GridItem
-                    name={localize('Austria')}
```

Comment:
Then you can remove `localize` functions from these props and just pass `TString` to the **name** property